launcher/archive.py

```diff
@@ -33,7 +33,8 @@ def get_mime_from_file(filename) -> str:
 
 def _7zip_extract(f: str, p: str) -> None:
     if run(['7z', 'x', '-y', f'-o{p}', f]).returncode != 0:
-        raise SevenZipDecompressionError(f'7z error will decompressing {f}')
+        if run(['unrar', '-y ','x', f'{f}', f'{p}']).returncode != 0:
```

Comment:
This will use unrar instead of 7z for all archives (win32) and rar / 7z+bcj2 for *nix.

Reply:
It will use unrar for all archives that fail to extract.

Reply:
I'm thinking about checking if filename contains .rar and then only use unrar if it's true

Reply:
Creating a new function to fix correctly #127 (and reverting workaround #128) will be better